projects/packages/image-cdn/tests/php/test_class.image_cdn.php

```diff
@@ -792,6 +793,26 @@ public function test_image_cdn_return_custom_size_array_dimensions() {
 		$this->helper_remove_image_sizes();
 	}
 
+	/**
+	 * Tests Photon image_downsize will return a cropped image for custom size if the custom size matches a registered size.
+	 *
+	 * @author kraftbj
+	 * @covers Image_CDN::filter_image_downsize
+	 * @since $$next-version$$
+	 */
+	public function test_image_cdn_return_custom_size_array_uses_registered_crop() {
+		$test_image = $this->helper_get_image();
+
+		// Declaring the size array directly, registered size jetpack_cropped_size of 165 by 165, cropped.
+		$this->assertEquals(
+			'resize=165%2C165',
+			$this->helper_get_query( Image_CDN::instance()->filter_image_downsize( false, $test_image, array( 165, 165 ) ) )
+		);
```

Comment:
Would be great to also test edge cases and their expected results:

`null,null`, `'null', 'null'`, `165'`, `'165'`, `true`, `9000,9000`, etc.

Reply:
A bit OOS and not really necessary as the filter only looks for preexisting image sizes. But, I have added a null test case anyway.